settings.json

```diff
@@ -1,22 +1,26 @@
 {
   "bot_cmd": "!bot",
-  "api_token": "",
+  "api_token": "MjExNDMyOTUxNzY4OTQwNTQ0.Czkv7g.EVaOok96HO7BNC15Ezth511FyN0",
```

Comment:
Whoops... :grin: 
You should remove this and revoke your token over at discord, bro.